shared.py

```diff
@@ -1,2 +1,43 @@
+import yaml
+import sys
+
```

Comment:
```suggestion
import argparse
```

Import argparse as well